CHANGELOG.md

```diff
@@ -13,6 +13,24 @@
   - `register` and `register_from_file` on class `snowflake.snowpark.udtf.UDTFRegistration`
   - `register` and `register_from_file` on class `snowflake.snowpark.stored_procedure.StoredProcedureRegistration`
   - `udf`, `udtf` and `sproc` in `snowflake.snowpark.functions`
+- Support `Column` as an input argument to a stored procedure call
```

Comment:
To keep the bullets symmetrical, we could change this to say:

Added support for \`Column` as an input argument to a stored procedure call.